libraries/app.rb

```diff
@@ -105,11 +125,15 @@ def post_install
                   command "sudo installer -pkg #{install_dir}/Contents/Resources/Packages/#{pkg} -target /"
                 end
               end unless xcode_packages.nil?
+            end
+            only_if { ::Dir.exist?(install_dir) }
```

Comment:
The original has this `only_if` check on "/Applications/Xcode.app" ... but not sure even that's correct because it runs the `xcode_packages` every time.